application/views/quickTranslation/translatetabs_view.php

```diff
@@ -65,8 +68,8 @@ class='ajax-loader' alt='<?php eT("Loading..."); ?>'/>
     </div>
     <?php
     if (isset($singleTabFieldsData)) {
-        if ($singleTabFieldsData[0]['all_fields_empty']) : ?>
-            <p><?php eT("Nothing to translate on this page"); ?></p><br/>
+        if ($allFieldsEmpty) : ?>
+            <p><?php eT("Nothing to translate on this page.."); ?></p><br/>
```

Comment:
why was this copy changed? you added two dots at the end, which will invalidate every existing translation of this. Please change it back.

Reply:
Nice catch. Sorry.